src/components/plots/areaPlot.ts

```diff
@@ -26,13 +26,32 @@ export module Plots {
       this.attr("stroke", defaultColor);
     }
 
+    public y(): Plots.AccessorScaleBinding<number, number>;
+    public y(y: number | Accessor<number>): Area<X>;
+    public y(y: number | Accessor<number>, yScale: QuantitativeScale<number>): Area<X>;
+    public y(y?: number | Accessor<number>, yScale?: QuantitativeScale<number>): any {
+      if (y == null) {
+        return super.y();
+      }
+      super.y(y, yScale);
```

Comment:
Oh I see.  In this specific circumstance this should be ok.  But in the other scenarios there are now incorrect castings